### PR TITLE
Add styles to Changed class

### PR DESCRIPTION
**Pull request by shikhanansi**
@lnewson please review and stamp.

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -147,34 +146,35 @@ class: changelog
...
-* Fixed long toolbar groups extending outside of the editor instead of wrapping
+* Fixed long toolbar groups extending outside of the editor instead of wrapping.
 * Fixed dialog titles so they are now the proper case.
-* Fixed color picker default to be #000000 instead of #ff00ff.
-* Fixed "match case" option on the Find and Replace dialog is no longer selected by default.
-* Fixed vertical alignment of toolbar icons #DES-134
-* Fixed toolbar icons not appearing on IE11 #DES-133
+* Fixed color picker default to be `#000000` instead of `#ff00ff`.
+* Fixed "match case" option on the **Find and Replace** dialog is no longer selected by default.
+* Fixed vertical alignment of toolbar icons `#DES-134`.
```
Comment (lnewson) [inline changelog.md]: Not related to this change, but `#DES-134` and `#DES-133` below shouldn't be in here (they are JIRA numbers). Not sure how that was missed :disappointed:

File: changelog.md [hunk 2]
```diff
@@ -10,54 +10,54 @@ class: changelog
...
-* Added missing fixed_toolbar_container setting has been reimplemented in the Silver theme.
-* Added a new toolbar setting `toolbar_drawer` that moves toolbar groups which overflow the editor width into either a `sliding` or `floating` toolbar section.
-* Updated the build process to include package lock files in the dev distribution archive.
-* Fixed inline dialogs did not have aria attributes.
-* Fixed default icons are now available in the UI registry, allowing use outside of toolbar buttons.
+* Added missing `fixed_toolbar_container` setting that has been reimplemented in the Silver theme.
+* Added a new toolbar setting `toolbar_drawer` that moves toolbar groups which overflow the editor width into either a *sliding* or *floating* toolbar section.
+* Changed the build process to include package lock files in the dev distribution archive.
+* Fixed inline dialogs that did not have aria attributes.
+* Fixed default icons, now available in the UI registry, to allow use outside of toolbar buttons.
```
Comment (lnewson) [inline changelog.md]: This doesn't quite sound right. I can't quite say what's wrong but it seems to change the original meaning. Maybe `Fixed default icons not being available in the UI registry, to allow use outside of toolbar buttons.` or something? The key point of this change, is that the default icons weren't in the registry and now they are. The last part is just useful information.

File: changelog.md (hunk 1 above)
Comment (shikhanansi) [reply changelog.md]: Removed JIRA numbers

File: changelog.md (hunk 2 above)
Comment (shikhanansi) [reply changelog.md]: Reworded sentence for technical accuracy.

**Pull request merged**